src/locales/fr/translation.json

```diff
@@ -1,115 +1,70 @@
 {
-  "menu": {
```

Comment:
These changes look unintentional.

We should also document the best flow for adding translation strings:
- change or add an English string and merge the PR
- wait 1 day for it to sync to transifex
- translate in transifex
- download the json files and make PR

related to: https://github.com/oasisprotocol/oasis-wallet-web/pull/810#issuecomment-1103830049